js/private/js_info_files.bzl

```diff
@@ -17,9 +17,9 @@ def _js_filegroup_impl(ctx):
         include_npm_linked_packages = ctx.attr.include_npm_linked_packages,
     ))
 
-js_filegroup = rule(
+js_info_files = rule(
```

Comment:
Normally "JsInfo" is one word, like "filegroup" and many other things. But our rules already start with `js_*` so 🤷

Reply:
I think the JsInfo camel case translates over to js_info fine. There is already a `js_info` helper function so this will be consistent with that.